contracts/FileStorage.sol

```diff
@@ -199,8 +205,9 @@ contract FileStorage is AccessControlEnumerableUpgradeable {
 
     function deleteFile(string calldata filePath) external {
         address owner = msg.sender;
-        ContentInfo memory file = getContentInfo(owner, filePath);
+        (ContentInfo memory file, DetailedInfo memory details) = getContentInfo(owner, filePath);
```

Comment:
It's cheaper here not to load whole DelatiledInfo into memory but use
`DetailedInfo storage details` instead.